content/en/blog/2025/expose-otel-collector-gateway-api/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
+date: 2025-04-6
+author: >
+  [Vipin Vijaykumar](https://github.com/vipinvkmenon)
+sig: OpenTelemetry JS
```

Comment:
```suggestion
sig: OpenTelemetry JS
```

Based on #6502, this is not the sponsoring SIG, either @open-telemetry/sig-end-user-approvers, but I also think there has to be support from @open-telemetry/operator-approvers and @open-telemetry/collector-approvers

Reply:
Have updated to End User SIG

Reply:
Anything to add in the sig?